androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/RouteIcon.kt

```diff
@@ -11,8 +11,8 @@ import com.mbta.tid.mbta_app.model.RouteType
 @Composable
 fun routeIcon(routeType: RouteType) =
     when (routeType) {
-        RouteType.BUS -> Pair(painterResource(id = R.drawable.mode_bus), "Bus")
-        RouteType.COMMUTER_RAIL -> Pair(painterResource(id = R.drawable.mode_cr), "Commuter Rail")
-        RouteType.FERRY -> Pair(painterResource(id = R.drawable.mode_ferry), "Ferry")
-        else -> Pair(painterResource(id = R.drawable.mode_subway), "Subway")
+        RouteType.BUS -> Pair(painterResource(id = R.drawable.mode_bus), null)
+        RouteType.COMMUTER_RAIL -> Pair(painterResource(id = R.drawable.mode_cr), null)
+        RouteType.FERRY -> Pair(painterResource(id = R.drawable.mode_ferry), null)
+        else -> Pair(painterResource(id = R.drawable.mode_subway), null)
```

Comment:
These have no screen reader text on iOS, and consequently there is no existing translation for these (although lowercase "bus" and "ferry" are translated as vehicle types for screen reader text), so I'm just removing these here for now. If I read things right, the usages of this are route pills (which should probably specify their own alt text when using these icons for shuttle or ferry routes) and vehicle pucks (which are inherently visual presentations of that information), so this is probably close enough to acceptable that we can fix it at QA time if we need to.